node-postgres/run-app.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+
+CURRENT_DIR=`dirname $0`
+ARTIFACTS_PATH=`realpath $CURRENT_DIR`
+YUGABYTE_HOME_DIRECTORY=/home/ec2-user/code/yugabyte-binary/yugabyte-2.19.0.0
```

Comment:
Why do you need yugabyte version 2.19.0.0 hard-coded? The pipeline will take the one which is configured for the build.

Reply:
This was required for local testing, have removed them.